Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -107,6 +107,12 @@ static int CompareGameListItems(const GameListItem* iso1, const GameListItem* is
 			if (iso1->GetCountry() < iso2->GetCountry())
 				return -1 * t;
 			return 0;
+		case CGameListCtrl::COLUMN_FNAME:
+			if (iso1->GetFileName() > iso2->GetFileName())
+				return  1 * t;
+			if (iso1->GetFileName() < iso2->GetFileName())
+				return -1 * t;
+			return 0;
```

Comment:
You can use `strcasecmp` here, like the code for `COLUMN_MAKER` and `COLUMN_ID`.